### PR TITLE
Adding dummy data for front end use

### PR DESCRIPTION
**Pull request by frenchal17**

### CODE REVIEW

File: SPARCore/GADArticle.h [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+@property (nonatomic, strong) NSArray *tags;
+@property (nonatomic, strong) NSString *brief;
+@property (nonatomic, strong) NSDate *datePublished;
+@property (nonatomic, strong) NSString *content;
+//@property (nonatomic, strong) NSSOMETHING *headerImage;
+@property (nonatomic, strong) NSString *title;
+@property (nonatomic, strong) NSString *issue;
+@property (nonatomic, strong) NSString *articleId;
+
++ (NSArray <GADArticle *> *) loadArticles;
```
Comment (alexthemitchell) [inline SPARCore/GADArticle.h]: Can we get a better name? `sampleArticles` or `dummyArticles` is fine, but `loadArticles` implies we're doing something real.

File: SPARCore/GADArticle.m [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        article.brief = [NSString stringWithFormat:@"This is very brief %i", i];
+        article.content = @"This is much less brief because it should be longer than the brief";
+        article.datePublished = [NSDate date];
+        article.issue = @"ae2f3b54-a49f-11e6-b1ab-a0999b05c023";
+        article.articleId = @"db7bc676-ba7d-11e6-8de8-a20f19d048b3";
+    
+        [articleArray addObject:article];
+    }
+    return articleArray;
+
```
Comment (alexthemitchell) [inline SPARCore/GADArticle.m]: Errant whitespace

File: SPARCore/GADArticle.m [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+        article.datePublished = [NSDate date];
+        article.issue = @"ae2f3b54-a49f-11e6-b1ab-a0999b05c023";
+        article.articleId = @"db7bc676-ba7d-11e6-8de8-a20f19d048b3";
+    
+        [articleArray addObject:article];
+    }
+    return articleArray;
+    
+}
+
```
Comment (alexthemitchell) [inline SPARCore/GADArticle.m]: Errant Whitespace

Comment (alexthemitchell) [inline SPARCore/GADArticle.m]: One newline before `@end`, one after.

File: SPARCore/GADArticle.m (hunk 2 above)
Comment (alexthemitchell) [inline SPARCore/GADArticle.m]: No newline between `return` and close bracket.

File: SPARCore/GADArticle.m (hunk 3 above)
Comment (enathang) [inline SPARCore/GADArticle.m]: Remove this white space

File: SPARCore/GADArticle.m [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+        article.brief = [NSString stringWithFormat:@"This is very brief %i", i];
+        article.content = @"This is much less brief because it should be longer than the brief";
+        article.datePublished = [NSDate date];
+        article.issue = @"ae2f3b54-a49f-11e6-b1ab-a0999b05c023";
+        article.articleId = @"db7bc676-ba7d-11e6-8de8-a20f19d048b3";
+    
+        [articleArray addObject:article];
+    }
+    return articleArray;
+}
```
Comment (enathang) [inline SPARCore/GADArticle.m]: Will prob look better if you add 1-2 spaces before }

Will leave you to decide

File: SPARCore/GADArticle.m [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+
```
Comment (addigould) [inline SPARCore/GADArticle.m]: Do we need this whitespace line?

Comment (addigould) [inline SPARCore/GADArticle.m]: Do we need the last line to be whitespace?

**Pull request merged**